systemd/rollback.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Rollback Helper for Registration
 DefaultDependencies=no
-Requires=network-online.target
-After=local-fs.target network-online.target
+After=local-fs.target systemd-networkd-wait-online.service network.service
```

Comment:
We don't use systemd-networkd and network.service does not mean the network is online, so how do you make sure that we only run the service if the network is online?